sdcm/cluster_aws.py

```diff
@@ -417,9 +417,8 @@ def node_setup(self, node, verbose=False):
                                   server_encrypt=self._param_enabled('server_encrypt'),
                                   client_encrypt=self._param_enabled('client_encrypt'))
 
-            node.remoter.run('sudo yum install -y {}-gdb'.format(node.scylla_pkg()),
```

Comment:
Why did you remove scylla_pkg installation? Is it part of the image now?

Reply:
It's just installing the scylla-gdb package, I moved the installation to `node_setup() -> config_setup()`